### ISSUE TITLE
Minify svg create a file that don't load on a web

### ISSUE THREAD

**Issue by imaginabit** [lishu/vscode-svg2#10]
When I use minify SVG in a file get a svg that I can view without problems but when i put like 
<img src="mini.svg" > in a html file ,I try on firefox and chrome and the minified file fails and show img alt text

I test it with tiger.svg file on inkscape examples https://inkscape.org/~inkscape/%E2%98%85example-tiger

**Comment by lishu** [comment]
Sorry, I can't found where is wrong, Please get more information about it.
![SVGTEST-TIGER](https://user-images.githubusercontent.com/1331008/76478221-e7ee6180-6442-11ea-876a-ade6595de246.png)
I have Chrome 80.0.x and Firefox 73.0.x

BTW: Some Security reasons, you may not see svg in `file:` protocol.

**Comment by imaginabit** [comment]
I'm have the problem when i use with the img tag , using a local sever , cant upload to any place right now, and it works if I do like you show but when I use the img tag didn't works

`<img src="img/tiger-min.svg" alt="tigre" >` => shows tigre but no image

**Comment by lishu** [comment]
This is setting default is remove xmlns, you can `uncheck` this options to fix.
![A10](https://user-images.githubusercontent.com/1331008/76533967-d7c29a80-64b3-11ea-9d7f-b3ee778e64b0.png)

**Comment by lishu** [comment]
Close Reason: Solution given and customer no longer reply
